rdflib/parser.py

```diff
@@ -10,12 +10,13 @@
 
 """
 
+from typing import TYPE_CHECKING
 import codecs
-import os
```

Comment:
Why this change?  It seems to be causing the tests to fail.

Reply:
I think this PR was created before #1441 which is where use of the os module was re-introduced.

Reply:
Made a PR against this PR to fix: #1480